src/ctia/domain/entities.clj

```diff
@@ -74,6 +74,19 @@
       (contains? schema (s/required-key k))
       (contains? schema k)))
 
+(defn make-valid-time
+  "make a valid-time object from either in this order:
+  the newest value, the previous value or the default value"
+  [prev-valid-time new-valid-time now]
+  {:valid_time {:start_time
+                (or (:start_time new-valid-time)
+                    (:start_time prev-valid-time)
+                    now)
```

Comment:
`:start_time` and `:end_time` are optional in [CTIM](https://github.com/threatgrid/ctim/blob/master/src/ctim/schemas/common.cljc#L336). Why set a default value?

Reply:
That's what that piece of code used to do, but we can make it a little smarter, set a default value only if the key is required.